Gordon360/Controllers/RecIM/MatchesController.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Gordon360.Models.CCT;
+using Gordon360.Models.ViewModels.RecIM;
+using Gordon360.Services.RecIM;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Data.SqlClient;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace Gordon360.Controllers.RecIM
+{
+    [Route("api/recim/[controller]")]
+    [AllowAnonymous]
+    public class MatchesController : GordonControllerBase
+    {
+        private readonly IMatchService _matchService;
+
+        public MatchesController(IMatchService matchService)
+        {
+            _matchService = matchService;
+        }
+
+        /// <summary>
+        /// Not implemented (yet) might be removed
+        /// </summary>
+        /// <param name="day"></param>
+        /// <param name="active"></param>
+        /// <returns></returns>
+        /// <exception cref="NotImplementedException"></exception>
+        [HttpGet]
+        [Route("")]
+        public ActionResult<IEnumerable<MatchViewModel>> GetMatches([FromQuery] DateTime? day, bool active)
+        {
+            throw new NotImplementedException();
+        }
+
+        /// <summary>
+        /// Fetches Match by MatchID
+        /// </summary>
+        /// <param name="matchID"></param>
+        /// <returns></returns>
```

Comment:
```suggestion
        /// <returns>The match with the requested matchID (or null)</returns>
```

Reply:
Well it may have not been worded in the clearest way, but I am noting the possibility of a null return instead of a Match object. If you send a matchID and it doesn't match up with an object in the database, you would throw a 404: Not Found error and (implicitly) return null.

Reply:
Well I don't know about any 404, but it's true that you may return null, since the matchID may not match any Match in the database.